src/SWP/Bundle/FixturesBundle/Resources/themes/DefaultTheme/theme.json

```diff
@@ -10,6 +10,45 @@
             "role": "Organization"
         }
     ],
+   "settings": {
+        "primary_color": {
+            "label": "Primary Color",
+            "value": "#d74b4b",
+            "type": "string"
+        },
+
+        "primary_font_family": {
+            "label": "Primary Font Family",
+            "value": "Oswald",
+            "type": "string",
+            "help": "The primary font",
+            "options": [
+                {"value": "Oswald", "label": "Oswald"},
+                {"value": "Lato", "label": "Lato"},
+                {"value": "Roboto", "label": "Roboto"}
+            ]
+        },
+        "secondary_font_family": {
+            "value": "Tahoma",
+            "type": "string",
+            "options": [
+                {"value": "Tahoma", "label": "Tahoma"},
+                {"value": "Lato", "label": "Lato"},
+                {"value": "Roboto", "label": "Roboto"}
```

Comment:
@ljubar Would be nice to have fallback for fonts like sans-serif, just in case of google font error.